src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -1237,58 +1088,14 @@ def validate_by_direct_connection(
         telemetry_data: OCSPTelemetryData,
         hostname: str = None,
         do_retry: bool = True,
+        **kwargs,
```

Comment:
I like to add explicit Anys (even it they are implicitly there already) to show that typing was added and not forgotten.
```suggestion
        **kwargs: Any,
```